elastic/apm/track.json

```diff
@@ -0,0 +1,57 @@
+{
+  "version": 2,
+  "description": "Elastic APM benchmark for Rally",
+  "data-streams": [
+    {"name": "traces-apm*"},
+    {"name": "metrics-apm*"},
+    {"name": "logs-apm*"}
+  ],
+  "corpora": [
+    {
+      "name": "apm-events", 
+      "documents": [{
+        "base-url": "https://rally-tracks.elastic.co/apm",
+        "source-file": "documents.ndjson.bz2",
+        "document-count": 19557,
+        "uncompressed-bytes": 54917896,
+        "includes-action-and-meta-data": true
+      }]
+    }
+  ],
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "delete-data-stream",
+        "data-stream": [
+          "traces-apm*",
+          "metrics-apm*",
+          "logs-apm*"
+        ]
+      }
+    },
```

Comment:
This only deletes existing datastreams but not the templates, right?

Reply:
Yes, it deletes only the existing data streams. more about this operation can be found here: https://esrally.readthedocs.io/en/stable/track.html#delete-data-stream